core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -311,7 +311,9 @@ private[spark] object UIUtils extends Logging {
       id: Option[String] = None,
       headerClasses: Seq[String] = Seq.empty,
       stripeRowsWithCss: Boolean = true,
-      sortable: Boolean = true): Seq[Node] = {
+      sortable: Boolean = true,
+      // The tooltip information could be None, which indicates header does not have a tooltip.
+      tooltipHeaders: Seq[Option[String]] = Seq.empty): Seq[Node] = {
```

Comment:
How about using `Map[Int, String] = Map.empty`.

Reply:
@gengliangwang the other arguments like headers, headerClasses are Seqs. Why make this one a Map?
I think having Nones there makes it more explicit that it has to have the same length that the other Seqs. Otherwise it's easy to miss an error when adding a new column that all indices in the Map shift.

Reply:
It is the same case as headerClasses, I have done it as the same way to be homogeneous

Reply:
I see. My concern was the length of `tooltipHeaders` can be different with `headers`. Passing parameter `Seq[Option[String]]` seems equivalent to `Map[Int, String]`.
I am fine with either way.